README.md

```diff
@@ -394,6 +394,7 @@ Platform | Description | Environments
 :--|:--|:--
 **N9k**   | Support includes all N9xxx models  | bash-shell, guestshell

```

Comment:
Please update the copyright at the end of the file.

Reply:
Done